packages/package-is-installable/src/checkPlatform.ts

```diff
@@ -15,34 +16,36 @@ export default function checkPlatform (
   packageId: string,
   wantedPlatform: WantedPlatform
 ) {
-  const platform = process.platform
-  const arch = process.arch
-  let osOk = true
-  let cpuOk = true
+  const { platform, arch } = process
+  const currentLibc = getLibcFamilySync() ?? 'unknown'
```

Comment:
Is it fast to read it every time? It might be better to read it only once per installation, so move it out from the function.

Reply:
ok

Reply:
maybe i need to put `getLibcFamilySync()` function out the `package-is-installable` ? and pass the result to the function `checkPackage()` ?

Reply:
Or just i use a env like `process.env.libc` to keep the value at `package-is-installable`?

Reply:
i read the source code of `getLibFamilySync()`, it just get a process.report value and deal it. Just like the `process.platform` and `process.arch` here, so it might not be slow?